source/administration/config-settings.rst

```diff
@@ -4159,25 +4163,50 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"ExperimentalChannelSidebarOrganization": off`` with options ``off``, ``default_on`` and ``default_off``.                                         |
+| This feature's ``config.json`` setting is ``"ExperimentalChannelSidebarOrganization": disabled`` with options ``enabled`` and ``disabled``.                                         |
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Sidebar Organization (Experimental)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Enable Legacy Sidebar
+^^^^^^^^^^^^^^^^^^^^^
+
+*Not available in Mattermost Cloud. Available in Mattermost Server v5.32 and later.*
+
+This setting re-enables the legacy sidebar functionality for all users on this server. We strongly recommend that you leave this configuration setting disabled. The new sidebar offers most of the same functionality available in the legacy sidebar and includes a number of new features that make channel management easier.
+
+**Disabled**: The new sidebar functionality is enabled including custom collapsible channel categories, multi-channel select, drag, drop, and move channel organization, channel and category sorting, and more. This is the default setting.
+
+**Enabled**: When enabled, the legacy sidebar features are enabled for all users on this server. However, Mattermost users can’t disable legacy sidebar functionality in **Account Settings**.
+
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableLegacySidebar": disabled`` with options ``enabled``, and ``disabled``.                                                            |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Sidebar Organization
+^^^^^^^^^^^^^^^^^^^^
+
+This setting applies to the legacy sidebar only. You must enable the `Enable Legacy Sidebar <https://docs.mattermost.com/administration/config-settings.html#enable-legacy-sidebar>`__ configuration setting to see and enable this functionality in the System Console.
+
+.. note::
+
+  We strongly recommend that you leave the **Enable Legacy Sidebar** configuration setting disabled. The new sidebar offers most of the same functionality available in the legacy sidebar, and includes a number of new features that make channel management easier.
```

Comment:
This experimental setting is not recommended for production environments. The new channel sidebar matches and exceeds the feature set offered by this configuration setting.

We strongly recommend that you leave the **Enable Legacy Sidebar** configuration setting disabled so users can access new channel sidebar features, including custom, collapsible channel categories, drag and drop, unread filtering, channel sorting options, and more. Learn more about these features (link to docs page).

Reply:
Let's add this note to every one of the experimental settings that only work with the legacy sidebar ^